### ISSUE TITLE
Unable to Login into Database using Sql Server Username and Password.

### ISSUE THREAD

**Issue by abdullahamjad786** [DrJohnT/AzureDevOpsExtensionsForSqlServer#15]
Hi Everyone, 

I am using this Extension for Database Deployment Option. Actually i have to run queries on Azure Sql Databases. And those Databases don't support Windows Login. So i have to give Sql Server Username and Password. 
But i have searched a lot and i didn't find any way to add my credentials in this Task. Can anyone please help me out how i can do that? My work is stuck due to this. :(

**Comment by coldacid** [comment]
If I had known that this extension didn't support username/password authentication I wouldn't have even installed it. This is an absolute must-have.

**Comment by EwneoN** [comment]
Yes I really fail to see the point of releasing this without authentication options. Seriously, what real world SQL Server database is not behind authentication?

I think I myself will simply be butchering the PS scripts for my own purposes.

**Comment by DrJohnT** [comment]
I am working on this now.  Just added the capability to my other extension [RunSqlCmdScripts](https://marketplace.visualstudio.com/items?itemName=DrJohnExtensions.RunSqlCmdScripts).  Will let you know soonest
